### PR TITLE
Add affinity to Helm chart

### PR DESCRIPTION
**Pull request by dongheeJeong**
*Issue #, if available:*
Not available.

*Description of changes:*
Add affinity to Helm chart.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by goyalya** [comment]
Thank you for submitting the request. Could you please elaborate on the need to add Affinity option ?

**Comment by dongheeJeong** [comment]
@goyalya 

As EKS fargate does not support daemonset, affinity is required like [this](https://github.com/aws/amazon-vpc-cni-k8s/blob/85654cf9b02f7aed240fd5b2f43be19f47800109/config/master/aws-k8s-cni.yaml#L265-L268).

**Review by simonmarty — APPROVED** [review]

File: charts/secrets-store-csi-driver-provider-aws/templates/.daemonset.yaml.swp
Comment (danmancuso) [inline charts/secrets-store-csi-driver-provider-aws/templates/.daemonset.yaml.swp:1]: Remove this .swp file before merging.

**Review by danmancuso — CHANGES_REQUESTED** [review]

File: charts/secrets-store-csi-driver-provider-aws/templates/.daemonset.yaml.swp
Comment (danmancuso) [reply charts/secrets-store-csi-driver-provider-aws/templates/.daemonset.yaml.swp:1]: Removed.

**Review by danmancuso — APPROVED** [review]

**Pull request merged**